examples/src/main/java/com/expediagroup/sdk/xap/examples/scenarios/lodging/HotelIdsSearchEndToEndScenario.java

```diff
@@ -0,0 +1,352 @@
+    return hotelListingsResponse;
+  }
+
+  /**
+   * Reads given number of property ids from the file pointed by the download URL.
+   *
+   * @param downloadUrl The download URL of the zip file containing the property information.
+   * @return A list of property ids read from the file.
+   */
+  private List<String> getPropertyIdsFromListingsFile(String downloadUrl) {
+    List<String> propertyIds = new ArrayList<>();
+    HttpURLConnection connection = null;
+    try {
+      // Open a connection to the URL
+      URL url = new URL(downloadUrl);
+      connection = (HttpURLConnection) url.openConnection();
+      connection.setRequestMethod("GET");
+      connection.setDoInput(true);
+
+      try (ZipInputStream zipStream = new ZipInputStream(connection.getInputStream())) {
+        ZipEntry entry;
+        while ((entry = zipStream.getNextEntry()) != null) {
+          if (entry.getName().endsWith(".jsonl")) {
+            LOGGER.info("Reading property ids from file: {}", entry.getName());
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(zipStream))) {
+              String line;
+              ObjectMapper objectMapper = new ObjectMapper();
+              while ((line = reader.readLine()) != null
+                  && propertyIds.size() < SAMPLE_ITEMS_RESTRICTION) {
+                // Parse the property id from the json object
+                // An example json line from the jsonl file:
+                /*
+                {
+                  "propertyId": {
+                    "expedia": "1234567",
+                    "hcom": "123456789",
+                    "vrbo": "123.1234567.7654321"
+                  },
+                  "bookable": {
+                    "expedia": true,
+                    "hcom": true,
+                    "vrbo": true
+                  },
+                  "propertyType": {
+                    "id": 16,
+                    "name": "Apartment"
+                  },
+                  "lastUpdated": "10-27-2024 13:41:16",
+                  "country": "France",
+                  "inventorySource": "vrbo",
+                  "referencePrice": {
+                    "value": "89.52",
+                    "currency": "USD"
+                  },
+                  "vrboPropertyType": {
+                    "instantBook": true
+                  }
+                }
+                */
+                JsonNode jsonNode = objectMapper.readTree(line);
+                // Check if the property is accessible from Lodging Listings API
+                // (Vrbo properties that are not instantBookable are not accessible for now)
+                if (!jsonNode.get("propertyId").get("vrbo").asText().isEmpty()
+                    && jsonNode.has("vrboPropertyType")
+                    && !jsonNode.get("vrboPropertyType").get("instantBook").asBoolean()
+                ) {
+                  // Skip the property if it is a not instant bookable Vrbo property
```

Comment:
If it is **not an** instant bookable Vrbo property

Reply:
Fixed